webhook/webhook.go

```diff
@@ -0,0 +1,103 @@
+package webhook
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/DzyubSpirit/reTGanslatorBot/bot"
+	_ "github.com/GoogleCloudPlatform/functions-framework-go/funcframework"
+	"github.com/GoogleCloudPlatform/functions-framework-go/functions"
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+)
+
+var (
+	botHandler      *bot.Handler
+	botWebhookToken string
+)
+
+func init() {
+	functions.HTTP("UpdateHandler", updateHandler)
+}
+
+func init() {
+	botWebhookToken = os.Getenv("WEBHOOK_TOKEN")
+
+	botToken := os.Getenv("BOT_TOKEN")
+	if botToken == "" {
+		log.Fatalf("BOT_TOKEN has to be specified")
+	}
+
+	path := "serverless_function_source_code/config.json"
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		// Fall back to the current working directory if that file doesn't exist.
+		path = "config.json"
+	}
+
+	configFile, err := os.Open(path)
+	if err != nil {
+		log.Fatalf("Failed to open config.json: %v", err)
+	}
+
+	configBytes, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		log.Fatalf("Failed to read config.json: %v", err)
+	}
+
+	var config bot.Config
+	if err = json.Unmarshal(configBytes, &config); err != nil {
+		log.Fatalf("Failed to parse config.json: %v", err)
+	}
+
+	tgBot, err := tgbotapi.NewBotAPI(botToken)
+	if err != nil {
+		log.Fatalf("Bot API failed to initialize: %v", err)
+	}
+
+	botHandler = bot.NewHandler(config, tgBot)
+
+	tgBot.Debug = true
+
+	log.Printf("Authorized on account %s", tgBot.Self.UserName)
+}
+
+func updateHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		log.Printf("Unknown HTTP method: %s", r.Method)
```

Comment:
"Unkown HTTP method" can be used as a text for the error instead of just giving the generic text via httpErr

Reply:
I am wondering whether we need these errors at all? tg api does not require it, doesn't it? :thinking:

Reply:
You're right, I just thought that it would be a nice thing to do if, for example, we would have a prober for monitoring the bot :)